src/main/kotlin/no/nav/familie/tilbake/api/dto/ManuellBrevmottakerDto.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.familie.tilbake.api.dto
+
+import no.nav.familie.kontrakter.felles.tilbakekreving.Vergetype
+import no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker.domene.MottakerType
+import java.util.UUID
+
+data class ManuellBrevmottakerDto(
+    val id: UUID? = null,
+    val type: MottakerType,
+    val navn: String,
+    val adresselinje1: String,
+    val adresselinje2: String?,
+    val postnummer: String,
+    val poststed: String,
+    val landkode: String,
+    var vergetype: Vergetype? = null
+)
```

Comment:
I ef-sak så setter vi aldri manuell adresse på en brevmottakere, vi velger kun en ident (person eller org)

Jeg lurer egentlige på om modellen her og i [kontrakter](https://github.com/navikt/familie-kontrakter/blob/master/felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/Brevmottaker.kt#L12) burde være noe i stil med at man kan sette type,ident og navn (påkrev) men så kan man legge inn manuell adresse hvis ønskelig.

Litt som modellen er for `DistribuerJournalpostRequest` i kontrakter, der adressen er optional 👀

Reply:
Høres ut som en godt idè, men ident trenger kanskje ikke være påkrevd, da det ikke er helt avklart enda om det skal være nødvendig å sette ident i ba-sak

Reply:
Men vi kan sikkert ha en validering som sjekker at den er satt hvis manuell adresse ikke er satt